init-tools.sh

```
@@ -64,6 +64,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

In that case, please fail the build instead of continuing ahead.